src/core/curlify.js

```diff
@@ -25,15 +25,18 @@ export default function curl( request ){
   if ( headers && headers.size ) {
     for( let p of request.get("headers").entries() ){
       let [ h,v ] = p
-      type = v
       curlified.push( "-H " )
       curlified.push( `"${h}: ${v}"` )
+      if (isMultipartFormDataRequest) {
```

Comment:
Why do we have this condition here? If the request is considered as multipart request I still want to process other headers and not ignore them. This condition effectively does that. Is that something you intended? If yes, why ?

Reply:
@char0n This is intentional. After initialization, we only want to set this flag once (to true). If you don't `break`, it will overwrite again, possibly as false, which then fails test.

Reply:
Regarding the break stetement: https://github.com/swagger-api/swagger-ui/pull/6152/files#r442378491

You're right in the fact that signal variable will be overridden to false by subsequent loop. Good catch!
But this implementation will stop the loop for good and curlified array will not be filled with additional headers that needs transformation to curl form.
IMHO correct implementation without introducing additional problems would be :

```js
isMultipartFormDataRequest = isMultipartFormDataRequest || /^content-type$/i.test(h) && /^multipart\/form-data$/i.test(v)
```